doc/source/tutorials/iiasa_dbs.ipynb

```diff
@@ -84,23 +84,24 @@
      "text": [
       "INFO:root:You are connected to the iamc15 scenario explorer. Please cite as:\n",
       "\n",
-      "D. Huppmann, E. Kriegler, V. Krey, K. Riahi, J. Rogelj, S.K. Rose, J. Weyant, et al., IAMC 1.5°C Scenario Explorer and Data hosted by IIASA. IIASA & IAMC, 2018. doi: 10.22022/SR15/08-2018.15429, url: data.ene.iiasa.ac.at/iamc-1.5c-explorer\n"
+      "D. Huppmann, E. Kriegler, V. Krey, K. Riahi, J. Rogelj, S.K. Rose, J. Weyant, et al., IAMC 1.5C Scenario Explorer and Data hosted by IIASA. IIASA & IAMC, 2018. doi: 10.22022/SR15/08-2018.15429, url: data.ene.iiasa.ac.at/iamc-1.5c-explorer\n"
      ]
     }
    ],
    "source": [
     "df = pyam.read_iiasa_iamc15(\n",
     "    model='MESSAGEix*', \n",
     "    variable=['Emissions|CO2', 'Primary Energy|Coal'], \n",
-    "    region='World'\n",
+    "    region='World',\n",
+    "    meta=['category']\n",
     ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Here we pulled out all results for model(s) that start with 'MESSAGEix' that are in the 'World' region and associated with the two named variables.\n",
+    "Here we pulled out all results for model(s) that start with 'MESSAGEix' that are in the 'World' region and associated with the two named variables. We also added the \"category\" metadata, which tells us which kind of scenario each is in SR1.5 parlance.\n",
```

Comment:
suggest to rephrase to "which tells us the climate impact categorisation of each scenario as assessed in the IPCC SR15."